iotdb/src/main/java/org/apache/iotdb/db/auth/authorizer/BasicAuthorizer.java

```diff
@@ -238,6 +240,7 @@ public void start() throws StartupException {
     try {
       init();
     } catch (AuthException e) {
+      logger.error(e.getMessage());
```

Comment:
I suggest to add some simple info about this error, instead of just e.getMessage()

Reply:
Agreed, and the stack trace is important for debugging